lab_1/main.py

```diff
@@ -125,4 +211,44 @@ def sort_concordance(tokens: list, word: str, left_context_size: int, right_cont
     left_sort = True
     --> [['dog', 'is', 'happy', 'but', 'the', 'cat'], ['man', 'is', 'happy', 'the', 'dog', 'is']]
     """
-    pass
+    conco_all = []
+    index_word = 0
+    left_context_bool, right_context_bool = False, False
+    if isinstance(left_sort, bool):
+        if checker(tokens, list) and checker(word, str) \
+                and checker(left_context_size, int) and checker(right_context_size, int):
+
+            condition_to_sort = (left_context_size < 0 and left_sort is True) or \
```

Comment:
No need for explicit check, 
Just use `and left_sort`

Reply:
condition_to_sort is that condition that we've discussed. 
If you mean checker for ints, without it I have a bug later.